app/controllers/products_controller.rb

```diff
@@ -1,10 +1,35 @@
 class ProductsController < ApplicationController
   def index
+    @products = Product.all
   end
 
   def new
+    @product = Product.new
+  end
+
+  def create
+    @product = Product.create(product_params)
```

Comment:
@amitshethe Add validations in models & handle if any error is raised how we can show error.
For create & update both

Reply:
@amitshethe Above is incorrect, if you use Product.create then it itself can save data or raise errors.
So there is no point in adding product.save.
Instead use assign_attributes method to assign the attrs at the start instead of create and then product.save will do rest.